css-positioning/css-positioning-quiz-notes.md

```diff
@@ -0,0 +1,48 @@
+# css-positioning-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is the **default** value for the `position` property of HTML elements?
+  A: static
+- How does setting `position: relative` on an element affect document flow?
+  A: the element can be shifted from its normal position using top, right, bottom, left but stays within the document flow
+- How does setting `position: relative` on an element affect where it appears on the page?
+  A: it can be shifted around
+- How does setting `position: absolute` on an element affect document flow?
+  A: it is removed entirely from the flow
+- How does setting `position: absolute` on an element affect where it appears on the page?
+  A: its positioned relative to its nearest positioned ancestor.
+- How do you constrain an absolutely positioned element to a containing block?
+  A: position fixed
```

Comment:
To constrain an absolutely position element, you would give the containing block `position: relative`, or any `position` value other than `static`